jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/URIUtil.java

```diff
@@ -1779,6 +1779,25 @@ public static List<URI> split(String str)
         return uris;
     }
 
+    /**
+     * Optionally wrap a URI with `jar:file:` (zipfs) syntax if the input URI
+     * is an archive and isn't wrapped.
+     *
+     * @param uri the input URI
+     * @return the input URI optionally wrapped with `jar:` if it's an archive.
+     * @see FileID#isJavaArchive(Path)
+     * @see URIUtil#toJarFileUri(URI)
+     */
+    public static URI wrapArchiveToJarFileUri(URI uri)
```

Comment:
Doesn't `toJarFileUri()` already provide the exact same functionality?

Reply:
Almost, it the `toJarFileURI()` will return null if the input URI is not a java archive, it's a subtle difference.

I wonder if we can just adjust `toJarFileURI` instead?

Reply:
URIUtil.wrapArchiveToJarFileUri has been removed.